
From: <a.othieno@bluewin.ch> (Arthur Othieno)

ChangeSet 1.1371.413.23 [1] introduced the file
include/asm-generic/dma-mapping-broken.h for architectures that don't
support the new DMA API. I don't know if this is the case with s390,
however, ChangeSet 1.1371.445.6 [2] introduced a set of changes that
duplicate those in include/asm-generic/dma-mapping-broken.h.

This patch squishes that duplication by simply including
include/asm-generic/dma-mapping-broken.h in
include/asm-s390/dma-mapping.h.

Signed-off-by: Arthur Othieno <a.othieno@bluewin.ch>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-s390/dma-mapping.h |   13 +------------
 1 files changed, 1 insertion(+), 12 deletions(-)

diff -puN include/asm-s390/dma-mapping.h~s390-use-include-asm-generic-dma-mapping-brokenh include/asm-s390/dma-mapping.h
--- 25/include/asm-s390/dma-mapping.h~s390-use-include-asm-generic-dma-mapping-brokenh	2004-07-26 16:26:08.754807200 -0700
+++ 25-akpm/include/asm-s390/dma-mapping.h	2004-07-26 16:26:08.757806744 -0700
@@ -9,17 +9,6 @@
 #ifndef _ASM_DMA_MAPPING_H
 #define _ASM_DMA_MAPPING_H
 
-static inline void *dma_alloc_coherent(struct device *dev, size_t size,
-			 dma_addr_t *dma_handle, int flag)
-{
-	BUG();
-	return 0;
-}
-
-static inline void dma_free_coherent(struct device *dev, size_t size,
-		       void *vaddr, dma_addr_t dma_handle)
-{
-	BUG();
-}
+#include <asm-generic/dma-mapping-broken.h>
 
 #endif /* _ASM_DMA_MAPPING_H */
_
